guide/inheritance-wallet/wallet-creation.md

```diff
@@ -0,0 +1,338 @@
+
+##### Bob adds the first two keys
+Bob gets started and imports the first two keys by himself. One from his personal Trezor and one from a Coldcard that he shares with Alice.
+
+{% include image-gallery.html pages = page.images_primary-keys-bob %}
+
+##### Bob requests the third key from Alice
+Next, it's time to add Alice's key, which will be imported from her Bitbox. Our app makes this process easy, because Bob can just create a key request that Alice can scan with her phone. Alternatively, he can also send it to her via a secure channel.
+
+{% include image-gallery.html pages = page.images_primary-keys-request-alice %}
+
+##### Alice imports the key from her signing device
+Alice has also downloaded the app, so she taps the big plus button on the homescreen and selects "provide a key" from the menu that pops up. On the next screen, she chooses to scan the key request. After doing that, the app displays the wallet configuration. Alice sees that her key will be used in the primary key set. She taps "provide key" and the app takes her through the same process like Bob. 
+
+{% include image-gallery.html pages = page.images_primary-keys-alice-export-key %}
+
+##### Alice provides her key to Bob
+But there is one additional step to take: Alice needs to transfer the information about her key back to Bob, so that he can add it to the wallet configuration. Since they are in the same room together, Bob scans the QR code that is shown on Alice's screen and goes through the same process as for the first two keys. 
+
+{% include image-gallery.html pages = page.images_primary-keys-alice-add-to-wallet %}  
+
+##### Time to hit pause
+All primary keys have now been added to the wallet. Next, Bob and Alice need to add the inheritance keys in the same way. They plan to do that during a family gathering which is coming up on the following weekend. For now, Bob hits "Save and finish later". The application saves the progress locally and allows Bob to continue later.
+
+{% include image-gallery.html pages = page.images_creation-save %}
+
+### Adding the inheritance keys
+
+Bob and Alice already met with Edward, their lawyer, ahead of the family meeting and imported his key to the wallet. So only Christina's and David's keys remain to be added to the wallet.
+
+During the family reunion, Bob opens our application and resumes the wallet creation process. Just like with Alice's key, he creates a key request for Christina and David.
+
+##### Christina creates a new key in the app
+Christina could follow the same procedure for importing a key from her existing signing device, as described above. However, Christina does not want to use her existing hardware wallet from the family savings. Instead, she wants to create a new key for the family savings wallet, so that she can keep it on a seperate signing device. 
```

Comment:
```suggestion
Christina could follow the same procedure for importing a key from her existing signing device, as described above. However, Christina does not want to use her existing hardware wallet from the family savings. Instead, she wants to create a new key for the family savings wallet, so that she can keep it on a separate signing device. 
```